controllers/deadmanssnitchintegration/deadmanssnitchintegration_controller.go

```diff
@@ -330,21 +330,17 @@ func (r *DeadmansSnitchIntegrationReconciler) createSnitch(dmsi *deadmanssnitchv
 				return err
 			}
 
-			var snitch dmsclient.Snitch
 			if len(snitches) <= 0 {
 				newSnitch := dmsclient.NewSnitch(snitchName, dmsi.Spec.Tags, "15_minute", "basic")
 				newSnitch.Notes = fmt.Sprintf("cluster_id: %s\nrunbook: https://github.com/openshift/ops-sop/blob/master/v4/alerts/cluster_has_gone_missing.md", clusterID)
 				// add escaping since _ is not being recognized otherwise.
 				newSnitch.Notes = "```" + newSnitch.Notes + "```"
 				logger.Info(fmt.Sprint("Creating snitch:", snitchName))
-				snitch, err = dmsc.Create(newSnitch)
+				_, err = dmsc.Create(newSnitch)
```

Comment:
would this cause the snitches to be re-created for restart situations? (like operator-restart/cycling)